src/query/api/v1/httpd/handler.go

```diff
@@ -129,6 +139,24 @@ func (h *Handler) RegisterRoutes() error {
 	return nil
 }
 
+func (h *Handler) m3AggServiceOptions() *placement.M3AggServiceOptions {
+	maxResolution := time.Duration(0)
+	for _, ns := range h.clusters.ClusterNamespaces() {
```

Comment:
Want to do a nil check here?  Just in case `m3dbClusters` wasn't specified?

Reply:
sure